cmd/keeper/cmd/keeper.go

```diff
@@ -142,6 +143,7 @@ func init() {
 
 	CmdKeeper.PersistentFlags().BoolVar(&cfg.canBeMaster, "can-be-master", true, "prevent keeper from being elected as master")
 	CmdKeeper.PersistentFlags().BoolVar(&cfg.canBeSynchronousReplica, "can-be-synchronous-replica", true, "prevent keeper from being chosen as synchronous replica")
+	CmdKeeper.PersistentFlags().BoolVar(&cfg.skipLockFile, "skip-lock-file", false, "prevent keeper to skip lock file. Warning! It'll cause data corruptions if two keepers are concurrently running with the same data dir.")
```

Comment:
I'll propose to call it `disable-data-dir-locking` since this is the behavior and in future how it's implemented may change.

Also `prevent keeper to skip lock file` is quite confusing, What about something like "disable locking on data dir. Warning .... "

Reply:
@sgotti  Thanks for the suggestion I did the push to the pull request with "disable-data-dir-locking"